eslint-config-base/package.json

```diff
@@ -1,20 +1,20 @@
 {
   "name": "@macpaw/eslint-config-base",
-  "version": "1.4.1",
+  "version": "2.0.0-alpha.4",
   "main": "index.js",
   "scripts": {
     "version:patch": "yarn version --patch --no-git-tag-version",
     "version:minor": "yarn version --minor --no-git-tag-version",
     "version:major": "yarn version --major --no-git-tag-version"
   },
   "dependencies": {
-    "eslint-plugin-import": "^2.23.4"
+    "eslint-plugin-import": "^2.26.0"
   },
   "devDependencies": {
-    "eslint": "^7.31.0"
+    "eslint": "^8.16.0"
   },
   "peerDependencies": {
-    "eslint": "^6.0.0 || ^7.0.0"
+    "eslint": "^6.0.0 || ^7.0.0 || ^8.0.0"
```

Comment:
how about to drop 6 and 7?

Reply:
sounds ok to me

Reply:
@korywka Added to release 2.0.0-alpha.5

Reply:
Why not support at least two latest versions? Especially if it doesn't require additional maintenance from us. Some projects might still have the previous Eslint version and no team's capacity to update it.

Reply:
No team capacity doesn't mean it will stop to work. My suggestion was to force to upgrade projects with team capacity. Plus avoid braking changes in the near future (e.g. `eslint-plugin-import` in this PR doesn't work with 7.0-7.2 versions).

Reply:
In this case, that teams can use v1.4.1 as long as they want. I thought, it might be a good way to "gently push" teams to update dependencies. 
But you are right - it is a bit strange to drop support if we don't need anything to do to just continue.

Reply:
Ok. Makes sense if it makes the maintenance more complicated.

PS: I suppose the case like we have with the `eslint-plugin-import` is the reason why some packages keep eslint plugins in the peer dependencies.

Reply:
So it's the time to decide if we will do the same with peerdeps.

Reply:
I vote for supporting only v8

Reply:
And what about to move `eslint-plugin-import` to peer deps? @okovpashko

Reply:
@korywka  If we do so it will be correct to move the other plugins to the `peerDependencies` and it looks like not the thing we should do right now.

@hraboviyvadim  I found that `eslint-plugin-import` supports Eslint v7 starting from `7.2.0` so if there are no other plugins that require at least v8 I'd leave support for `^7.2.0 || ^8`.

Reply:
imho it doesn't looks OK if we search for min versions of our direct deps and tweak ours. it is easy while we have 1 dependency but not scaleable approach.

Reply:
I agree that might be not easy. But otherwise, it might be confusing if we claim that we support the particular version of the peer dependency or even Node.js itself but then NPM throws warnings or errors during the installation 🤷‍♂️

Reply:
to summarise - what cons will we get if move to v8 only? 
I see only one - it will force all teams to update deps and it may take some time.

Reply:
_gently reminder_ and not force for sure 🙃 nothing will brake.

Reply:
@korywka if set `peerDependencies` to `^8` you will get an npm error during installation if you don't have it. 

<img width="716" alt="image" src="https://user-images.githubusercontent.com/3851732/170984936-4435c66e-715e-4c6c-ac98-c94b0845a50e.png">

Or I just missed your point.

Reply:
Yep. Starting from version 8 NPM has stricter validation of the peer dependencies.

Reply:
To me it's ok.
Yes, in this case we won't get all projects updated tomorrow, it will take some time. 
But it's totally ok imho.

Reply:
@klianchina @lequanvn89 guys check this discussion pls and tell what you think

Reply:
I agree with @okovpashko, it would be great to keep the two latest versions. 
Not critical, but it can irritate us a little just because updating eslint on all projects can take more time than we expect

Reply:
@klianchina on the other hand, if we leave support for version 7, there will be no motivation to update our local eslint packages for a very long time 😀

Reply:
Maybe if I have no other motivation, I don`t need it?)))
But ok ok, let it be

Reply:
To be serious, I see 2 options here:
- drop everything old and support only ESLint@8
- set peerDependencies as `^7.2.0 || ^8`

Looks like we some kind of poll. Maybe TechRadar would also help xD @onikiienko

Reply:
So let it be :) Review pls this PR